pkg/reconciler/tenancy/clusterworkspacedeletion/clusterworkspace_deletion_controller.go

```diff
@@ -248,9 +258,23 @@ func (c *Controller) finalizeWorkspace(ctx context.Context, workspace *tenancyv1
 		if workspace.Finalizers[i] == deletion.WorkspaceFinalizer {
 			workspace.Finalizers = append(workspace.Finalizers[:i], workspace.Finalizers[i+1:]...)
 
+			lcn := logicalcluster.From(workspace)
```

Comment:
Note: we typically name this type of variable `clusterName` (although we're not 100% consistent in the codebase)